legacy/application/models/ShowBuilder.php

```diff
@@ -283,9 +283,9 @@ private function makeScheduledItemRow($p_item)
             $formatter = new LengthFormatter(Application_Common_DateHelper::secondsToPlaylistTime($run_time));
             $row['runtime'] = $formatter->format();
 
-            $row['title'] = htmlspecialchars($p_item['file_track_title']);
-            $row['creator'] = htmlspecialchars($p_item['file_artist_name']);
-            $row['album'] = htmlspecialchars($p_item['file_album_title']);
+            $row['title'] = $p_item['file_track_title'] ? htmlspecialchars($p_item['file_track_title']) : '';
+            $row['creator'] = $p_item['file_artist_name']] ? htmlspecialchars($p_item['file_artist_name']) : '';
```

Comment:
```suggestion
            $row['creator'] = $p_item['file_artist_name'] ? htmlspecialchars($p_item['file_artist_name']) : '';
```

Reply:
@mp3butcher this is what is causing the failure

Reply:
...i'm tired....